server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1255,9 +1265,54 @@ public void deleteConsumer(
                 returnCerts.add(cert);
             }
         }
+        // we want to insert the content access cert to this list
+        if (consumer.getContentAccessCert() != null) {
+            try {
+                returnCerts.add(contentAccessCertService.getCertificate(consumer));
+            }
+            catch (IOException ioe) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate."));
+            }
+            catch (GeneralSecurityException gse) {
+                throw new BadRequestException(i18n.tr("Cannot retrieve content access certificate."));
+            }
+        }
         return returnCerts;
     }
 
+    @ApiOperation(notes = "Retrieves the body of the Content Access Certificate for the Consumer",
+        value = "getContentAccessBody")
+    @ApiResponses({ @ApiResponse(code = 404, message = "") })
+    @GET
+    @Path("{consumer_uuid}/content")
+    @Produces(MediaType.APPLICATION_JSON)
+    public ContentAccessListing getContentAccessBody(
+        @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
+        @QueryParam("if-modified-since") Date since) {
```

Comment:
I found in smoketesting work in candlepin/python-rhsm#191, that by default, it appears to use `java.util.Date.parse` (which accepts a very specific format of dates - and *not* ISO format); it would be great if this could just accept ISO dates, so that I can simply use `isoformat()` on a `datetime` object (or even just pass back in the string I get from the `/content` endpoint) instead of [this](https://github.com/candlepin/python-rhsm/blob/8e148184d99a271784c6503e2cc10527c1bafa16/src/rhsm/connection.py#L1097). Methinks this could be accomplished with a `StringParameterUnmarshaller` for `Date`? Also a quick note: there is some variance allowed in ISO-8601, in Python you get this variant:

    2016-11-17T17:00:00.123456-04:00

(ideally we should accept any ISO-8601 variant).